keyboards/handwired/dactyl_promicro/dactyl_promicro.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+/*
+#ifdef KEYBOARD_handwired_dactyl_promicro
+    #include "6x6.h"
+#endif*/
+
+//void promicro_bootloader_jmp(bool program);
+#include "quantum.h"
+
+
+#ifdef USE_I2C
+#include <stddef.h>
+#ifdef __AVR__
+	#include <avr/io.h>
+	#include <avr/interrupt.h>
+#endif
+#endif
+
+
+#define LAYOUT_6x6(\
+	L00, L01, L02, L03, L04, L05,                    R00, R01, R02, R03, R04, R05, \
+	L10, L11, L12, L13, L14, L15,                    R10, R11, R12, R13, R14, R15, \
+	L20, L21, L22, L23, L24, L25,                    R20, R21, R22, R23, R24, R25, \
+	L30, L31, L32, L33, L34, L35,                    R30, R31, R32, R33, R34, R35, \
+	L40, L41, L42, L43, L44, L45,                    R40, R41, R42, R43, R44, R45, \
+		 L51, L52, L53, L54, L55,                    R50, R51, R52, R53, R54      \
```

Comment:
Is this accurate? 

Wouldn't it be more like: 
```c
                            L55,L56,   R50,R51,                          \
                                L54,   R52,                              \
                        L53,L52,L51,   R55,R54,R53                      \
```

Reply:
well, the matrix shows how it is wired not how it is physically placed on the board.
can change that to reflect the key placement on the board if wanted.

Reply:
The LAYOUT macros are specifically intended to be a physical layout-to-matrix transform. Not actually using the real physical layout (or an approximation) is rather missing the point.

Reply:
no problem, will change that ;)